prelude/prelude.coma

```diff
@@ -67,6 +67,10 @@ module Opaque
   type ptr
   let fresh_ptr (ret (result : ptr) {true}) = any
 
+  function thin (p : ptr) : ptr
+  let thin (p : ptr) (ret (result : ptr) { result = thin p }) = any
```

Comment:
This `let` declaration is unnecessary and should go away. Logical functions can be called in Coma code, and this ends up in much shorter Coma code.